core/src/main/resources/hudson/PluginManager/table.jelly

```diff
@@ -96,11 +93,15 @@ THE SOFTWARE.
                   name="${p.displayName}"
                   data-plugin-id="${h.xmlEscape(p.name)}">
                   <l:isAdmin>
-                    <td align="center" data="${isUpdates ? null : it.unscientific(p.popularity)}">
```

Comment:
Previously, `data` allowed users to restore the default sort order (by popularity) after changing it. This looks like it is no longer possible now.

Reply:
From what I can see `table.jelly` is only ever used for the updates page so `isUpdates` is always `true`, so there's never actually any meaningful sorting.

Reply:
Interesting, it used to be for "updates" and "available", but that seems no longer the case.

https://github.com/jenkinsci/jenkins/blob/dacf078913d52a2e8c8a03f63d8401738ca317a7/core/src/main/resources/hudson/PluginManager/available.jelly#L64-L66 is probably a leftover from that heritage.